WordPressKit.xcodeproj/project.pbxproj

```diff
@@ -1942,7 +1942,6 @@
 			buildActionMask = 2147483647;
 			files = (
 				32A29A1F236BE4CC009488C2 /* post-autosave-mapping-success.json in Resources */,
-				93AC8EC61ED32FD000900F5A /* stats-v1.1-alltime.json in Resources */,
 				7403A2FC1EF06FEB00DED7DC /* me-settings-change-lastname-success.json in Resources */,
```

Comment:
not sure why this was removed here?

Reply:
`pod install` gives a warning for unknown file references -

[!] `<PBXSourcesBuildPhase UUID=`9368C7761EC5EF1B0092CE8E`>` attempted to initialize an object with an unknown UUID. `9368C7B41EC630270092CE8E` for attribute: `files`. This can be the result of a merge and  the unknown UUID is being discarded.

[!] `<PBXResourcesBuildPhase UUID=`9368C7821EC5EF1B0092CE8E`>` attempted to initialize an object with an unknown UUID. `93AC8EC61ED32FD000900F5A` for attribute: `files`. This can be the result of a merge and  the unknown UUID is being discarded.

These errors are too small for their own pod version update, but fit well for this changeset.